src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Button from './Button'
+const grey = 'rgb(224, 224, 224)'
+function ButtonPanel (props) {
+  return (
+    <div id='button-panel'>
+      <div className="row">
+        <Button name='AC' color={grey} clickHandler={buttonName => props.clickHandler(buttonName)}/>
```

Comment:
This is another time that instead of doing this:

```javascript
clickHandler={buttonName => props.clickHandler(buttonName)}
```
We should just be:

```javascript
clickHandler={prop.sclickHandler}
```

Again, try changing this without making `handeClick` method in `App.js` an arrow function and see what error you will get. :)